### PR TITLE
Latest translated strings

### PR DESCRIPTION
**Pull request by jhenshall**

### CODE REVIEW

File: locales/es.json [hunk 1]
```diff
@@ -1,5 +1,110 @@
 {
     "header": {
-        "getInvolved": "Spanish"
-    }
+        "getInvolved": "Involucrarse",
+        "data": "Datos",
+        "about": "Acerca de"
+    },
+    "homepage": {
+    	"page-title": "MapSwipe/Cada            ayuda a colocar familias en el mapa",
```
Comment (Hagellach37) [inline locales/es.json:8]: these spaces look a bit weird. maybe they have to be removed directly in transifex?

File: locales/es.json [hunk 2]
```diff
@@ -1,5 +1,110 @@
...
+    	"data-type": "Tipo de datos",
+    	"data-description": "Descripción del conjunto de datos",
+    	"projects-name": "Visión general de los proyectos",
+    	"projects-description": "Información acerca de los proyectos",
+    	"geometry-name": "Proyectos con Geometría",
+    	"geometry-description": "Información acerca de todos los proyectos con el proyecto geometría",
+    	"centroid-name": "Proyectos con Centroid",
+    	"centroid-description": "Proyecto Centroid",
+    	"leaderboard-title": "Tablero de dirección",
+    	"leaderboard-text": "                                           y crear una competencia amigable ? Desea saber donde está Ud enun mar de alrededor de 30.000 otros mapeadores?Hemos lanzado recientemente el tablero de directivas , permitiendo a los usuarios como tú ver la distancia que has recorrido y el número de objetos que has encontrado.Utiliza la función Búsqueda  para ubicar el nombre de usuario y poder comenzar.",
```
Comment (Hagellach37) [inline locales/es.json:64]: here again. not sure why this is an issue here again

File: locales/es.json [hunk 3]
```diff
@@ -1,5 +1,110 @@
...
+    	"leaderboard-link": "Encuentra más",
+    	"license-title": "Nuestra Licencia",
+    	"license-text1": "Este proyecto es parte de la comunidad OpenStreetMap. El objetivo son datos de alta calidad geográfica , de acceso libre y disponibles para todos. La licencia recíproca de OSM protege los datos de ser apropiados por servicios que no comparten con OMS",
+    	"license-text2": "Mapswipe es lanzada bajo una licencia \"liberal\" no recíproca (                              )Esta solamente requiere que los usuarios reconozcan la fuente. Ud puede hacer lo que quiera con los datos. Solo asegúrese de acreditar a los miembros de MapSwipe."
```
Comment (Hagellach37) [inline locales/es.json:68]: is there something hidden here in the brackets?

**Comment by jhenshall** [comment]
Thanks for the comments @Hagellach37 - we're working on this simultaneously. Closing this pull request without merging because hopefully this will come from Transifex now.

**Pull request closed without merging**